src/test/fuzz/util.h

```diff
@@ -250,6 +250,24 @@ template <class T>
     return false;
 }
 
+/**
+ * Returns a copy of the value selected from the given vector `v`.
+ */
+template <typename T>
+[[nodiscard]] T PickValue(FuzzedDataProvider& fuzzed_data_provider, const std::vector<T> v) noexcept
+{
+    assert(!v.empty());
+    return v[fuzzed_data_provider.ConsumeIntegralInRange<size_t>(0, v.size() - 1)];
+}
+
+/**
+ * Sets errno to a value selected from the given vector `errnos`.
+ */
+inline void SetFuzzedErrNo(FuzzedDataProvider& fuzzed_data_provider, const std::vector<int> errnos) noexcept
```

Comment:
same

Reply:
Also I am wondering why this can't use PickValueInArray

Reply:
1. Thanks! Fixed.
2. Now using `std::array` and `PickValueInArray`.